config.rb

```diff
@@ -63,6 +63,7 @@ def self.test?
   override :smtp_tls, true, bool
 
   # Override -- value is returned or the set default.
+  override :base_url, "http://localhost:9292", string
```

Comment:
We have hardcoded `https://console.ubicloud.com` strings in the codebase. Could you replace them as well?